pom.xml

```diff
@@ -44,7 +44,7 @@
         <commons.text.version>1.9</commons.text.version>
         <commons.lang.version>3.12.0</commons.lang.version>
         <commons.cli.version>1.4</commons.cli.version>
-        <mockito.version>3.9.0</mockito.version>
+        <mockito.version>3.10.0</mockito.version>
```

Comment:
Any chance we can find a way to avoid having to review changes like these? It is a bit cluttery... ;-)

Reply:
Good point - I'll ask Aaron how he handles these

Reply:
Aaron says I should use dependabot to do the pom updates, then I can merge those changes without others needing to review. I will add that as a new task